src/commonMain/kotlin/fr/acinq/lightning/channel/ChannelData.kt

```diff
@@ -350,23 +350,29 @@ data class LocalParams(
     val htlcMinimum: MilliSatoshi,
     val toSelfDelay: CltvExpiryDelta,
     val maxAcceptedHtlcs: Int,
-    val isInitiator: Boolean,
+    val isChannelOpener: Boolean,
```

Comment:
This ensures that every usage of that field is correctly updated (so that the compiler helps find them all). This is particularly useful when rebasing to correctly catch potential changes!